core/primitives/src/signature.rs

```diff
@@ -22,3 +22,33 @@ pub fn default_signature() -> Signature {
     let sig = [0u8; sodiumoxide::crypto::sign::ed25519::SIGNATUREBYTES];
     sodiumoxide::crypto::sign::ed25519::Signature(sig)
 }
+
+pub trait GroupSignature {
+    fn verify(&self, data: &[u8]) -> bool;
+    fn signer_count(&self) -> usize;
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone, Default)]
+pub struct DumbGroupSignature {
+    signatures: Vec<(PublicKey, Signature)>,
```

Comment:
I understand this is a mock, but for `validate_signature` interface this doesn't make much sense - public keys for validation should be passed from outside (e.g. Authority will provide them).